components/budget/filters/PeriodFilter.js

```diff
@@ -119,7 +123,7 @@ const TriggerContainer = styled(StyledButton)`
 const PeriodFilter = ({ onChange, value, inputId, ...props }) => {
   const [dateInterval, setDateInterval] = React.useState(() => getDefaultState(value));
   const setDate = (type, date) => {
-    setDateInterval(value => ({ ...value, [type]: !date ? null : normalizeDate(new Date(date)) }));
+    return setDateInterval(value => ({ ...value, [type]: !date ? null : normalizeDate(new Date(date), type) }));
```

Comment:
Why this return? The return value doesn't seem to be used

Reply:
Yes, we can remove it.